files/image_config/platform/rc.local

```diff
@@ -318,6 +318,11 @@ if [ -f $FIRST_BOOT_FILE ]; then
         # Initialize the SONiC's grub config
         mv /host/grub.cfg /host/grub/grub.cfg
     fi
+    sed -i 's/[^M] quiet/ crashkernel=768M quiet/' /host/grub/grub.cfg
```

Comment:
Please check whether 768M is absolutely necessary, 256M could be more reasonable?